test/esinstall/polyfill-node/polyfill-node.test.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const {install} = require('../../../esinstall/lib');
+
+describe('polyfill node', () => {
+  it('is enabled through polyfillNode', async () => {
+    const cwd = __dirname;
+    const dest = path.join(cwd, 'test-polyfill-node');
+    const spec = 'node-builtin-pkg';
+
+    const {
+      importMap: {imports}
+    } = await install([spec], {
+      cwd,
+      dest,
+      polyfillNode: true
+    });
+
+    expect(imports[spec]).toBeTruthy();
```

Comment:
I think this is one place where I really would like to test that the polyfill is being applied, and not assume that it’s working because there’s not a build error.

In the generated file, I think just adding a `expect(jsFileContents).toEqual(expect.stringContaining(…))` for a single line or snippet from the polyfill code would work—just anything that wasn’t in the original file to begin with.

Reply:
Ok, I've updated to ensure the `path.dirname` implementation is included.

Reply:
Thank you! Also you can blame me if this fails because it keeps changing. But I agree with the part you chose—seems safe 🤞🏻